pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricStreams.java

```diff
@@ -37,7 +37,7 @@ public class PrometheusMetricStreams {
      * @param value value of the sample
      * @param labelsAndValuesArray varargs of label and label value
      */
-    void writeSample(String metricName, Number value, String... labelsAndValuesArray) {
+    public void writeSample(String metricName, Number value, String... labelsAndValuesArray) {
```

Comment:
Why change here ?

Reply:
`writeSample()` method will be called by `addCustomizedMetrics()`, the default access permissions cannot be called.